messages/en.json

```diff
@@ -31,7 +31,9 @@
         "PING! {nick}",
         "I'm sorry Dave, I'm afraid I can't do that.",
         "Give me a ping, Vasily. One ping only, please.",
-        "Would you people please leave me alone? Seriously."
+        "Would you people please leave me alone? Seriously.",
+        "Did you know the plural of lykos is lykoi?",
```

Comment:
While entertaining, lykos is not in fact a hardcoded bot nick so this wouldn't make any sense on other networks. As such, it should be removed.